enhancements/unified-networking/README.md

```
@@ -0,0 +1,766 @@
+|----------|--------------|
+| ComputeInstance | `vm` |
+| Cluster (VM nodes) | `vm` |
+| Cluster (BM nodes) | `bm` |
+| BaremetalInstance | `bm` |
+
+#### Provider Setup (common)
+
+1. Provider deploys hosting cluster and fabric manager
+2. Provider creates NetworkClass for the region
+3. Provider creates PublicIPPool:
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)